lib/Onyx.js

```diff
@@ -62,11 +63,12 @@ const getSubsetOfData = (sourceData, selector) => (_.isFunction(selector)
  * The resulting collection will only contain items that are returned by the selector.
  * @param {Object} collection
  * @param {String|Function} selector (see method docs for getSubsetOfData() for full details)
+ * @param {Object} [withOnyxInstanceProps]
```

Comment:
Same as [this comment](https://github.com/Expensify/react-native-onyx/pull/247/files#r1186211659) should we update API.md to include that new param?